openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -9,18 +10,44 @@ class IntegrateKitsuNote(pyblish.api.ContextPlugin):
     order = pyblish.api.IntegratorOrder
     label = "Kitsu Note and Status"
     families = ["render", "kitsu"]
+
+    # status settings
     set_status_note = False
     note_status_shortname = "wfa"
 
+    # comment settings
+    CustomCommentTemplate = {}
+    CustomCommentTemplate["enabled"] = False
+    CustomCommentTemplate["comment_template"] = "{comment}"
+
+    def safe_format(self, msg, **kwargs):
+        def replace_missing(match):
```

Comment:
Must be outside of the function. With docstring

Reply:
Is that really a most?
Currently the `replace_missing()` function access the `safe_format()` function's `**kwargs`
How would `re.sub(pattern, replace_missing, msg)` pass the `**kwargs`? As the `msg` needs to be a string if reading the re.sub's docstring.

Any suggestions?

Reply:
> Currently the replace_missing() function access the safe_format() function's **kwargs
> How would re.sub(pattern, replace_missing, msg) pass the **kwargs?

Yes, I'd say so.

> Is that really a must?

However, for me personally I'm not really to bothered by it being inline - it's like a lambda function but then PEP8 defined as a function. It seems you both are mostly in the Kitsu code so I'll leave it up to you. Adding a docstring I'd recommend however.

Reply:
My bad, I don't know how, but I missed the use of `kwargs`... Keep it inside, but add docstring.